packages/scrollable-video/README.md

```diff
@@ -0,0 +1,143 @@
+# scrollable-video
+
+![npm (latest)](https://img.shields.io/npm/v/@twreporter/scrollable-video/latest)
+![npm (rc)](https://img.shields.io/npm/v/@twreporter/scrollable-video/rc)
+
+## Usage
+
+### Data Schema
+
+```js
+/**
+ *
+ *
+ * @typedef {Object} Data
+ * @property {AppProps} appProps
+ * @property {Caption[]} captions
+ * @property {Theme} theme
+ * @property {Video} video
+ */
+
+/**
+ * @typedef {Object} AppProps
+ * @property {number} secondPerVh
+ * @property {CaptionsSetting} captionsSetting
+ */
+
+/**
+ * @typedef {Object} CaptionsSetting
+ * @property {string} xBoxAlign - The global horizontal alignment of the caption box. Must be one of 'left', 'center', or 'right'.
+ * @property {string} xPosition - The global horizontal position of the captions relative to viewport.. Example:
+      { xBoxAlign: 'left', xPosition: '50px' } means the position is 50 px from the left edge of viewport to right
+      { xBoxAlign: 'center', xPosition: '50px' } means the position is 50 px from the center of viewport to right
+      { xBoxAlign: 'right', xPosition: '8.7%' } means the position is 8.7% (related to the section width) from the right edge of viewport to left
+ * @property {string} yBoxAlign - The global vertical alignment. Must be one of 'bottom' or 'top'.
+ * @property {string} yPosition - The global vertical position of the captions relative to viewport. Example:
+      { yBoxAlign: 'top', xPosition: '50px' } means the position is 50 px from the left edge of viewport to right
```

Comment:
`xPosition: '50px'` -> `yPostion: '50px'`?
`left edge of viewport to right` -> `top edge of viewport to bottom` ?